pkg/server/deprovision_test.go

```diff
@@ -89,6 +99,13 @@ func TestDeprovision(t *testing.T) {
 				Async: true,
 			},
 		},
+		{
+			name: "deprovision returns valid planids",
```

Comment:
It’s not clear to me what this test case does

Reply:
@pmorie my bad, copy and paste on this second test. Sorry. I'll fix this today.